index.html

```diff
@@ -23,6 +23,97 @@ <h2>Software Developer</h2>
+        <h3>Multi-Post Stories</h3>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <ul class="stack-container">
+            <li class="border-right padding-right">css</li>
+            <li class="border-right padding-right">html</li>          
+            <li class="border-right padding-right">bootstrap</li>         
+            <li class="padding-right">Ruby</li>
+        </ul>
+        <button type="button" class="project-button">See Project</button>
+    </section> 
+    <section class="experience-container">        
+        <img src="./images/Icons/bg.jpg" alt="project image">
+        <h3>Multi-Post Stories</h3>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <ul class="stack-container">
+            <li class="border-right padding-right">css</li>
+            <li class="border-right padding-right">html</li>          
+            <li class="border-right padding-right">bootstrap</li>         
+            <li class="padding-right">Ruby</li>
+        </ul>
+        <button type="button" class="project-button">See Project</button>
+    </section> 
+    <section class="experience-container">        
+        <img src="./images/Icons/bg.jpg" alt="project image">
+        <h3>Multi-Post Stories</h3>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <ul class="stack-container">
+            <li class="border-right padding-right">css</li>
+            <li class="border-right padding-right">html</li>          
+            <li class="border-right padding-right">bootstrap</li>         
+            <li class="padding-right">Ruby</li>
+        </ul>
+        <button type="button" class="project-button">See Project</button>
+    </section> 
+    <section class="experience-container">        
+        <img src="./images/Icons/bg.jpg" alt="project image">
+        <h3>Multi-Post Stories</h3>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <ul class="stack-container">
+            <li class="border-right padding-right">css</li>
+            <li class="border-right padding-right">html</li>          
+            <li class="border-right padding-right">bootstrap</li>         
+            <li class="padding-right">Ruby</li>
+        </ul>
+        <button type="button" class="project-button">See Project</button>
+    </section> 
     </main>
```

Comment:
- [ ] The entire About myself section is missing and this is a major requirement, kindly work on it.

Reply:
The about myself section is on top, I think your doing the review based on a different template not template 4